tests/image/instance_segmentation/test_data.py

```diff
@@ -43,3 +46,31 @@ def test_image_detector_data_from_folders(tmpdir):
     data = next(iter(datamodule.predict_dataloader()))
     sample = data[0]
     assert sample[DataKeys.INPUT].shape == (128, 128, 3)
+
+
+@pytest.mark.skipif(not _IMAGE_AVAILABLE, reason="image libraries aren't installed.")
+@pytest.mark.skipif(not _COCO_AVAILABLE, reason="pycocotools is not installed for testing")
+def test_instance_segmentation_output_transform():
+
+    sample = {
+        DataKeys.INPUT: torch.rand(3, 224, 224),
+        DataKeys.PREDS: {
+            "bboxes": [
+                {"xmin": 10, "ymin": 10, "width": 15, "height": 15},
+                {"xmin": 30, "ymin": 30, "width": 40, "height": 40},
+            ],
+            "labels": [0, 1],
+            "masks": [
+                np.random.randint(2, size=(128, 128), dtype=np.uint8),
+                np.random.randint(2, size=(128, 128), dtype=np.uint8),
+            ],
+            "scores": [0.5, 0.5],
+        },
+        DataKeys.METADATA: {"size": (224, 224)},
+    }
+
+    output_transform_cls = InstanceSegmentationOutputTransform()
+    data = output_transform_cls.per_sample_transform(sample)
+
+    assert data["masks"][0].size() == (224, 224)
+    assert data["masks"][1].size() == (224, 224)
```

Comment:
Nice test! Thank you for adding this as well.

Reply:
Thank you for your comment :)